workflows/epitopeprediction.nf

```diff
@@ -113,16 +113,16 @@ workflow EPITOPEPREDICTION {
     // See the documentation https://nextflow-io.github.io/nf-validation/samplesheets/fromSamplesheet/
     // ! There is currently no tooling to help you write a sample sheet schema
 
-    INPUT_CHECK.out.reads
+    INPUT_CHECK.out.meta
                 .branch {
                     meta_data, input_file ->
-                        variant_compressed : meta_data.inputtype == 'variant_compressed'
+                        variant_compressed : meta_data.file_type == 'variant_compressed'
                             return [ meta_data, input_file ]
-                        variant_uncompressed :  meta_data.inputtype == 'variant'
+                        variant_uncompressed :  meta_data.file_type == 'variant'
```

Comment:
In my opinion `inputtype` still describes it better than file_type since we have the same file type for differnent input (`tsv` for variants and peptides).